nipype/algorithms/tests/test_auto_NormalizeProbabilityMapSet.py

```diff
@@ -11,7 +10,7 @@ def test_NormalizeProbabilityMapSet_inputs():
 
     for key, metadata in list(input_map.items()):
         for metakey, value in list(metadata.items()):
-            yield assert_equal, getattr(inputs.traits()[key], metakey), value
+            assert getattr(inputs.traits()[key], metakey) == value
```

Comment:
are these changes done in the tools/checkspecs.py file? If not, they will go away with ```make specs```.